### ISSUE TITLE
Html emails are shown in a tiny preview iframe even if there is a lot more space that the iframe could use

### ISSUE THREAD

**Issue by IchbinkeinReh** [nextcloud/mail#3696]
### Expected behavior
Html emails are shown in a preview iframe that uses the available space

### Actual behavior
Html emails are shown in a tiny preview iframe even if there is a lot more space that the iframe could use

![ichbinkeinreh de_cloud_index php_apps_mail_box_53_thread_181735](https://user-images.githubusercontent.com/12527468/94995552-5fcad780-059f-11eb-821c-aec94738158a.png)

Mail app
Mail app version: v1.5.0

Mailserver or service: Strato

Server configuration
Operating system: Ubuntu 20.04

Web server: Apache

Database: MariaDB

PHP version: 7.4

Nextcloud Version: 20.0.0

Client configuration
Browser: Chrome 81

Operating system: Ubuntu 20.04

**Comment by austin3410** [comment]
I'm seeing the same exact behavior on a fresh install, is there anyway to fix this?

**Comment by ChristophWurst** [comment]
Fix is at https://github.com/nextcloud/mail/pull/3697

**Comment by Northcode** [comment]
I'm still experiencing this in Mail 1.8.1, on nextcloud 20.0.7.
In both chromium 88 and firefox 85.
The iframe height is stuck at 150px

**Comment by ChristophWurst** [comment]
Open a new ticket. I assume your message is within a message thread, that is where it is expected and aligns with what other email solutions do. Only single email get full height.

**Comment by Northcode** [comment]
Well this is awkward, I can no longer reproduce this even with the same emails that gave me trouble 2 days ago. I have no idea what changed but it now seems to be working as expected. Sorry for the trouble.

**Comment by jamestomasino** [comment]
I ran into the same issue on a fresh install of Nextcloud and running Mail 1.10.1. I am using Firefox and noticed the page was giving me warnings about mixed content (some not over HTTPS). When I disabled protection and allowed the page to load mixed content the rendering went to the full size. However, when I open the network tab and inspect the resources being loaded everything seems to be HTTPS.

I'm not sure this is helpful and the issue is marked as closed, but if it creeps up again perhaps this will help future debugging somewhat.
